### PR TITLE
Remove a few static_cast<int> 

### PR DESCRIPTION
**Pull request by aritorto**

### CODE REVIEW

**Comment by aritorto** [comment]
jenkins build this serial please

File: opm/grid/cpgrid/CpGrid.cpp [hunk 1]
```diff
@@ -1528,11 +1528,11 @@ bool CpGrid::adapt(const std::vector<std::array<int,3>>& cells_per_dim_vec,
                    const std::vector<std::array<int,3>>& endIJK_vec)
 {
     // To do: support coarsening.
-    assert(static_cast<int>(assignRefinedLevel.size()) == current_view_data_->size(0));
+    assert( int(assignRefinedLevel.size()) == current_view_data_->size(0));
```
Comment (bska) [inline opm/grid/cpgrid/CpGrid.cpp:1531]: Please don't do this.  The `int()` cast is strictly worse, from a semantic point of view, than `static_cast<int>()`.  Yes, the latter is more verbose, but the verbosity does have a real benefit here.

**Comment by aritorto** [comment]
int() is gone now

**Comment by aritorto** [comment]
jenkins build this serial please

File: opm/grid/cpgrid/CpGrid.cpp (hunk 1 above)
Comment (aritorto) [reply opm/grid/cpgrid/CpGrid.cpp]: Thanks for your feedback, it's gone now.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**